File: android/analysis-java/app/src/main/java/eu/id3/face/samples/analysisjava/Parameters.java
package eu.id3.face.samples.analysisjava;

import android.hardware.camera2.CameraCharacteristics;

public class Parameters {
    public final static int detectorThreadCount = 4;
    public final static int detectorConfidenceThreshold = 70;
    public final static int yawMaxThreshold = 10;
    public final static int pitchMaxThreshold = 20;
    public final static int rollMaxThreshold = 10;

    public final static int eyeOcclusionMaxThreshold = 50;
    public final static int noseOcclusionMaxThreshold = 50;
    public final static int mouthOcclusionMaxThreshold = 30;

    public final static int hatMaxThreshold = 10;
    public final static int mouthOpenMaxThreshold = 35;
    public final static int smileMaxThreshold = 75;

    public final static int maxProcessingImageSize = 512;
    public final static int cameraType = CameraCharacteristics.LENS_FACING_FRONT;
}

File: android/analysis/app/src/main/java/eu/id3/face/samples/analysis/Parameters.kt
package eu.id3.face.samples.analysis

/**
 * Contains the parameters needed in the app. Values of these parameters can be changed and should not be seen as "the perfect" values.
 */
internal object Parameters {
    const val detectorThreadCount = 4
    const val detectorConfidenceThreshold = 70

    const val yawMaxThreshold = 10
    const val pitchMaxThreshold = 20
    const val rollMaxThreshold = 10

    const val eyeOcclusionMaxThreshold = 50
    const val noseOcclusionMaxThreshold = 50
    const val mouthOcclusionMaxThreshold = 30

    const val hatMaxThreshold = 10
    const val mouthOpenMaxThreshold = 35
    const val smileMaxThreshold = 75

    const val maxProcessingImageSize = 512
}
